### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by twasyl**
The changes introduce terminology updates to replace `master` by `controller` and remaining `slave` by `agent`.
This is lead by [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: docs/troubleshooting.adoc
Comment (aHenryJard) [inline docs/troubleshooting.adoc:353]: are you sure that this property file exists ? `jenkins-agent.xml`

Comment (twasyl) [reply docs/troubleshooting.adoc:353]: I checked the source code and some other resources bundle, and it seems this file is already mentioned and searched

Comment (aHenryJard) [reply docs/troubleshooting.adoc:353]: ok then.

**Review by aHenryJard — APPROVED** [review]

**Review by amuniz — APPROVED** [review]
Looks good (ES translation specifically).

File: src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties
Comment (rsandell) [inline src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties]: I don't speak Portuguese, but looks like you missed one.

File: src/main/resources/hudson/os/windows/Messages_es.properties
Comment (rsandell) [inline src/main/resources/hudson/os/windows/Messages_es.properties:21]: What about the actual `Messages.properties`?

File: src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties
Comment (twasyl) [reply src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties]: You're right. Fixed

File: src/main/resources/hudson/os/windows/Messages_es.properties
Comment (twasyl) [reply src/main/resources/hudson/os/windows/Messages_es.properties:21]: No changes were required in `Messages.properties` if it's what you mean :)

**Review by rsandell — APPROVED** [review]

**Comment by twasyl** [comment]
I don't have write access on the repo, so please don't wait for me to merge this one :) If someone could do it, would be awesome. Thanks!

**Review by MRamonLeon — APPROVED** [review]

**Comment by Saibamen** [comment]
This is the only change between previous version and 1.8: https://github.com/jenkinsci/windows-slaves-plugin/compare/windows-slaves-1.7...windows-slaves-1.8

Why did you release a new version without any bugfixes or dependency updates?

**Comment by aHenryJard** [comment]
> This is the only change between previous version and 1.8: [windows-slaves-1.7...windows-slaves-1.8](https://github.com/jenkinsci/windows-slaves-plugin/compare/windows-slaves-1.7...windows-slaves-1.8)
> 
> Why did you release a new version without any bugfixes or dependency updates?

Terminology changes are highly expected by several end users, you can either see that as a bug or a feature. In both cases a release is a good option. And anyway better release often and small no ?

**Pull request merged**
